prometheus/src/cluster_metrics.rs

```diff
@@ -52,5 +81,60 @@ pub fn write_cluster_metrics<W: io::Write>(
         },
     )?;
 
+    // Vote accounts information
+    for vote_account in vote_accounts.iter() {
+        write_metric(
+            out,
+            &MetricFamily {
+                name: "solana_node_last_vote_slot",
```

Comment:
Now that we are going to monitor multiple account, I think the `solana_node_*` series of metrics might become confusing ... on the one hand the identity of the instance itself is `solana_node_identity_public_key_info`, but we also have “nodes” that are not *this* node ...

Maybe we should call these accounts that we monitor `solana_validator_*`?